src/App.tsx

```diff
@@ -1,7 +1,37 @@
-import React from 'react';
+import { Component, KeyboardEvent } from 'react';
```

Comment:
You don't have to change imports for this task. You could just use React.Component in row 7:)

Reply:
![image](https://user-images.githubusercontent.com/101139968/188388549-c1c725cd-f4ca-4704-bc41-03e3d7731157.png)